### ISSUE TITLE
Feature Request: Bash Completion w/ install

### ISSUE THREAD

**Issue by Gidgidonihah** [drone/drone#1951]
[urfave/cli](https://github.com/urfave/cli#bash-completion) makes it fairly simple to implement bash completion. Adding a single line to `drone/main.go` and bundling the autocomplete script.

I'd love to see the installation options also include bash/zsh autocompletion where applicable.

**Comment by tboerger** [comment]
Maybe you should simply provide a pr for that ;)

**Comment by Gidgidonihah** [comment]
I intended to right after opening the issue. The single line addition is easy.

I was then going to tweak the installation instructions to talk about the `autocomplete.sh` file. I then realized the installation instructions aren't in the README, but are hosted on http://readme.drone.io/admin/ (the cli specifically at http://readme.drone.io/usage/getting-started-cli/ which is not linked from the readme)

On top of that, it'd be nice to include the the autocomplete file with the build and add it as part of the installation.

I'd still be happy to open the PR, but I figured it wouldn't be much use without the installation information in the right place.
